src/main/java/com/fastcampus/scheduling/_core/security/filter/AuthenticationFilterForLocal.java

```diff
@@ -0,0 +1,54 @@
+package com.fastcampus.scheduling._core.security.filter;
+
+import com.fastcampus.scheduling._core.errors.exception.FieldMissExceptionException;
+import com.fastcampus.scheduling._core.security.dto.SignInRequest;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.AuthenticationServiceException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Slf4j
+public class AuthenticationFilterForLocal extends UsernamePasswordAuthenticationFilter {
+
+    public AuthenticationFilterForLocal(AuthenticationManager authenticationManager) {
+        super.setAuthenticationManager(authenticationManager);
+    }
+    @Override
+    public Authentication attemptAuthentication(HttpServletRequest request, HttpServletResponse response) throws AuthenticationException {
+        UsernamePasswordAuthenticationToken authenticationToken;
+        if (!request.getMethod().equals("POST")) {
```

Comment:
```suggestion
        if (!request.getMethod().equals(HttpMethod.POST.name())) {
```
String을 enum으로 표현도 가능할꺼 같습니다.